util/asm_dev/armv8/p256/src/select.S

```diff
@@ -0,0 +1,165 @@
+//------------------------------------------------------------------------------------
+// Copyright (C) Amazon
+//------------------------------------------------------------------------------------
+// void select_w5(uint64_t *val, uint64_t *in_t, int index);
+    .arch armv8.4-a+crc+sve
```

Comment:
Why requiring this particular .arch?
I do not see crc and sve instructions used in the following code.
I think this directive can be removed.